src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java

```diff
@@ -606,6 +607,41 @@ public void testIsCqlLibraryNameChangedReturnsTrueForChange() {
     assertThat(output, is(true));
   }
 
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForNullSupplementalData() {
+    final Measure original = Measure.builder().supplementalData(Collections.emptyList()).build();
+    final Measure changed = original.toBuilder().supplementalData(Collections.emptyList()).build();
+    boolean output = measureUtil.isSupplementalDataChanged(changed, original);
+    assertThat(output, is(false));
+  }
+
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForOriginalNonNullSupplementalData() {
```

Comment:
Please rename. Test returns true for removed SDE.
```suggestion
  public void testIsSupplementalDataChanged_ReturnsTrueForRemovedSupplementalData() {
```